src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -171,18 +182,41 @@ export const getModuleDetailsUma = async (
             isDisputed: isDisputed,
             isSettled: result.settled,
             resolvedPrice: result.resolvedPrice,
-            proposalHash: proposalHash
+            proposalHash: proposalHash,
+            proposalTime: result.timestamp
```

Comment:
When testing on a few proposals that have been proposed, I am seeing `proposalTime` returned as undefined. I am not seeing anywhere this is causing issues but want to highlight.

Reply:
That's weird. Everything else looks fine? There should always be a timestamp associated with an OO request.

Reply:
Huh, I'm wrong, there actually isn't a `timestamp` in the `Request` object.
https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/optimistic-oracle/interfaces/OptimisticOracleV2Interface.sol#L72

Reply:
I removed it since it doesn't exist/we're not trying to use it, anyway.